class.jetpack.php

```diff
@@ -4161,6 +4163,58 @@ function plugin_action_links( $actions ) {
 		return array_merge( $jetpack_home, $actions );
 	}
 
+	/**
+	 * Adds the deactivation warning modal if there are other active plugins using the connection
+	 *
+	 * @return void
+	 */
+	public function deactivate_dialog() {
+		global $pagenow;
+		if ( 'plugins.php' === $pagenow && self::is_active() ) {
```

Comment:
Instead of relying on the global, what would you think about using the info passed with the action?

```suggestion
	 *
	 * @param string $hook The current admin page.
	 *
	 * @return void
	 */
	public function deactivate_dialog( $hook ) {
		if (
			'plugins.php' === $hook
			&& self::is_active()
		) {
```